content/docs/pipelines/tutorials/sdk-examples.md

```diff
@@ -1,35 +1,44 @@
 +++
 title = "Using the Kubeflow Pipelines SDK"
-description = "How to use the Kubeflow Pipelines API to upload a local file to create a new pipeline version"
-weight = 1
+description = "Examples that demonstrate how to use the Kubeflow Pipelines SDK"
+weight = 10
 +++
 
 This guide provides examples that demonstrate how to use the Kubeflow Pipelines SDK.
 
 ## Before you start
 
-To follow the example, you must have installed Kubeflow Pipelines SDK version 0.2.5 or higher. Use the following instructions to install the Kubeflow Pipelines SDK and check the SDK version.
-
-* Install the [Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/)
-* Run the following command to check the version of the SDK
-```
-pip list | grep kfp
-```
-The response should be something like this:
-```
-kfp                      0.2.5
-kfp-server-api           0.2.5
-```
+To follow the examples in this guide, you must have Kubeflow Pipelines SDK
+version 0.2.5 or higher installed. Use the following instructions to install
+the Kubeflow Pipelines SDK and check the SDK version.
+
+1. Install the [Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/)
+1. Run the following command to check the version of the SDK
+   ```
+   pip list | grep kfp
+   ```
+   The response should be something like this:
+   ```
+   kfp                      0.2.5
+   kfp-server-api           0.2.5
+   ```
 
 ## Examples
 
 Use the following examples to learn more about the Kubeflow Pipelines SDK.
 
-### Using the SDK to create a pipeline and a pipeline version
+### Creating a pipeline and a pipeline version using the SDK
+
+The following example demonstrates how to use the Kubeflow Pipelines SDK to
+create a pipeline and a pipeline version.
 
-The following example demonstrates how to use the Kubeflow Pipelines SDK to create a pipeline and a pipeline version.
+In this example, you:
 
-In the example, we first use KFP API client to create a pipeline from a local file. When the pipeline is created, a default pipeline version is created under it automatically. Then, we again use KFP API client to create a new pipeline version from a local file, and the new version will be under the pipeline that just get created.
+* Use `kfp.Client` to create a pipeline from a local file using the Kubeflow
+  Pipelines API. When the pipeline is created, a default pipeline version is
```

Comment:
Would it be worth removing the words "using the Kubeflow Pipelines API"? As a reader, I'm not sure how the API fits in, given that this page is all about the SDK. We'd need to give a longer description of the relationship between the two, and I'm not sure that's necessary.

Reply:
Yeah, I think it makes sense to remove that.